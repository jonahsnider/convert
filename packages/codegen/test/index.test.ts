import {expect, test} from 'vitest';
import {codegen} from '../src/index.js';

test('codegen', async t => {
	expect(
		await codegen({
			best: {imperial: [], metric: []},
			conversions: {},
			temperatureDifferences: {},
		}),
	).toMatchSnapshot();
});
