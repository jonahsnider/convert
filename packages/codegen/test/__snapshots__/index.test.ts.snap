// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`codegen 1`] = `
"export const conversions = {__proto__: null} as const;

export const bestUnits = {
	imperial: [],
	metric: [],
} as const;

export const temperatureDifferences = {__proto__: null} as const;
"
`;
